content/blog/2023-05-21-Fair-Ease-EuroScienceGateway/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Stronger together: FAIR-EASE and EuroScienceGateway join forces" 
+tease: "Seconde blog post presenting the outcome of a galaxy training session"
+authors: "Marie Jossé"
+source_blog: "https://eosc.eu/news/stronger-together-fair-ease-and-eurosciencegateway-join-forces"
```

Comment:
I need to check, but can it be that this tag will simply redirect the user to this page?

Reply:
Uh perhaps I thought it would just write down the link in the description